gemrb/core/Strings/String.h

```diff
@@ -109,6 +109,17 @@ void RTrim(STR& string, StringViewT<STR> chars = WHITESPACE_STRING_VIEW(STR))
 	}
 }
 
+template <typename STR>
+STR RTrimOf(const STR& string, StringViewT<STR> chars = WHITESPACE_STRING_VIEW(STR))
+{
+	auto pos = FindLastNotOf(string, chars);
```

Comment:
why not just implement in terms of `RTrim`?

```
template <typename STR>
 STR RTrimOf(STR string, StringViewT<STR> chars = WHITESPACE_STRING_VIEW(STR))
 {
    return RTrim(string, chars);
}
```

Reply:
I do realize that you aren't always needing to copy the entire string, but given how memory allocation works it's not really worth fussing over. The use case is going to have only a few bytes max that its loping off and ultimately the allocation will be the same.

Reply:
I think naming to `RTrimCopy` might be a better name too.

Reply:
Then an extra function like that is not needed at all, it would just be an alias.

Reply:
thats not true, it creates a copy unlike `RTrim`

Reply:
Can do. `RTrim` just doesn't return anything.